### ISSUE TITLE
Windows 3090 Convolutional 2D network crashing

### ISSUE THREAD

**Issue by dmayhem93** [pytorch/pytorch#46150]
## 🐛 Bug
Pytorch nightly has issues running convolutional 2D networks on 3090

## To Reproduce

1. Install pytorch-nightly with python 3.8
2. Try to run a Convolutional 2D network (VGG, etc) on GPU
3. Python exception is raised

Faulting application name: python.exe, version: 3.8.3150.1013, time stamp: 0x5efe0bf7
Faulting module name: cudnn64_8.dll, version: 6.14.11.6050, time stamp: 0x5f1694f4
Exception code: 0xc0000409
Fault offset: 0x0000000000018a7d
Faulting process id: 0x4b9c
Faulting application start time: 0x01d69f62b3d0588f
Faulting application path: C:\ProgramData\Anaconda3\python.exe
Faulting module path: C:\ProgramData\Anaconda3\lib\site-packages\torch\lib\cudnn64_8.dll
Report Id: 1c5185e2-fea7-4703-b26d-4c88cb63512f
Faulting package full name: 
Faulting package-relative application ID: 

## Expected behavior

Convolutional network should not crash python

## Environment

Please copy and paste the output from our
[environment collection script](https://raw.githubusercontent.com/pytorch/pytorch/master/torch/utils/collect_env.py)
(or fill out the checklist below manually).

You can get the script and run it with:
```
wget https://raw.githubusercontent.com/pytorch/pytorch/master/torch/utils/collect_env.py
# For security purposes, please check the contents of collect_env.py before running it.
python collect_env.py
```
Collecting environment information...
PyTorch version: 1.8.0.dev20201010
Is debug build: True
CUDA used to build PyTorch: 11.0
ROCM used to build PyTorch: N/A

OS: Microsoft Windows 10 Home
GCC version: Could not collect
Clang version: Could not collect
CMake version: version 3.18.2

Python version: 3.8 (64-bit runtime)
Is CUDA available: True
CUDA runtime version: Could not collect
GPU models and configuration: Could not collect
Nvidia driver version: Could not collect
cuDNN version: Could not collect
HIP runtime version: N/A
MIOpen runtime version: N/A

Versions of relevant libraries:
[pip3] gpytorch==1.2.0
[pip3] layer-to-layer-pytorch==0.3.0
[pip3] numpy==1.19.2
[pip3] numpydoc==1.1.0
[pip3] pytorch-lamb==1.0.0
[pip3] revtorch==0.2.4
[pip3] torch==1.8.0.dev20201010
[pip3] torchvision==0.8.0.dev20201010
[conda] blas                      1.0                         mkl  
[conda] cudatoolkit               11.0.221             h74a9793_0  
[conda] gpytorch                  1.2.0                    pypi_0    pypi
[conda] layer-to-layer-pytorch    0.3.0                    pypi_0    pypi
[conda] mkl                       2020.1                      216  
[conda] mkl-service               2.3.0            py38hb782905_0  
[conda] mkl_fft                   1.1.0            py38h45dec08_0  
[conda] mkl_random                1.1.1            py38h47e9c7a_0  
[conda] numpy                     1.19.2                   pypi_0    pypi
[conda] numpy-base                1.18.5           py38hc3f5095_0  
[conda] numpydoc                  1.1.0                      py_0  
[conda] pytorch                   1.8.0.dev20201010 py3.8_cuda110_cudnn8_0    pytorch-nightly
[conda] pytorch-lamb              1.0.0                     dev_0    <develop>
[conda] revtorch                  0.2.4                    pypi_0    pypi
[conda] torchvision               0.8.0.dev20201010      py38_cu110    pytorch-nightly
```

## Additional context

I fixed it by downloading cuDNN 8.0.4 and moving all the .dlls into the lib folder, where the original one is located. It appears the cudnn_cnn, audnn_ops, and cudnn_adv dlls are required but aren't part of the conda build for this version


cc @malfet @seemethere @walterddr @csarofeen @ptrblck @peterjc123 @maxluk @nbcsm @guyang3532 @gunandrose4u @smartcat2010 @mszhanyi

**Comment by xsacha** [comment]
Since Nvidia has just recently fixed this in the very latest version and 1.7.0-rc1 was already cut, it might not be available in next release. Bit disappointing.

**Comment by cristianPanaite** [comment]
Hello @dmayhem93,

I could not reproduce this error on the latest versions.

My env:

`PyTorch version: 1.13.0`
`CUDA used to build PyTorch: 11.7`

`OS: Microsoft Windows 11`
`GCC version: Could not collect`
`Clang version: Could not collect`
`CMake version: version 3.22.1`

`Python version: 3.9.13 `
`Is CUDA available: True`
`CUDA runtime version: 11.7.99`
`Nvidia driver version: 522.06`

`Versions of relevant libraries:`
`[pip3] mypy-extensions==0.4.3`
`[pip3] numpy==1.21.5`
`[pip3] numpydoc==1.4.0`
`[pip3] torch==1.13.0`
`[pip3] torchaudio==0.13.0`
`[pip3] torchvision==0.14.0`
`[conda] blas                      1.0                         mkl`
`[conda] mkl                      2.021.4.0           haa95532_640`
`[conda] mkl-service          2.4.0            py39h2bbff1b_0`
`[conda] mkl_fft                 1.3.1            py39h277e83a_0`
`[conda] mkl_random        1.2.2            py39hf11a4ad_0`
`[conda] numpy                 1.21.5           py39h7a0a035_3`
`[conda] numpy-base        1.21.5           py39hca35cd5_3`
`[conda] numpydoc           1.4.0            py39haa95532_0`
`[conda] pytorch                1.13.0          py3.9_cuda11.7_cudnn8_0    pytorch`
`[conda] pytorch-cuda       11.7                 h67b0de4_0    pytorch`
`[conda] pytorch-mutex     1.0                        cuda    pytorch`
`[conda] torchaudio            0.13.0                   pypi_0    pypi`
`[conda] torchvision           0.14.0                   pypi_0    pypi`

I tried with a VGG net from [this](https://pytorch.org/hub/pytorch_vision_vgg) official example. 

If you think issue still persist, please open new issue with details on how to reproduce it on latest version.
